.env

```diff
@@ -1,10 +1,13 @@
 BCX_URL=https://api-stage-bcx.pillarproject.io
 TX_DETAILS_URL=https://ropsten.etherscan.io/tx/
 NETWORK_PROVIDER=ropsten
+COLLECTIBLES_NETWORK=rinkeby
 SDK_PROVIDER=https://api-stage-core.pillarproject.io
 NOTIFICATIONS_URL=https://api-stage-notifications.pillarproject.io
 SIGNAL_SERVER_HOST=https://signal-stage.pillarproject.io
 INVESTMENTS_URL=https://api-stage-investments.pillarproject.io
 SENTRY_DSN=https://82eb3c51aa80408597cac8ae5c18f9d1@sentry.io/1294444
 BUILD_NUMBER=STAGING::_build_number_
 BUILD_TYPE=development
+OPEN_SEA_API=https://rinkeby-api.opensea.io/api/v1
+OPEN_SEA_API_KEY=_open_sea_api_key_
```

Comment:
Maybe we should leave it blank here? As you can't run circleci process locally

Reply:
Yeah, will do :)